guizero/Padding.py

```diff
@@ -0,0 +1,67 @@
+from tkinter import Label, StringVar
+from . import utilities as utils
+from .base import TextWidget
+
+class Padding(TextWidget):
```

Comment:
By inheriting from TextWidget a Padding will inherit all the properties of a widget with text. Need to check if a padding should support the attributes of a TextWidget.